Userbookmarks

cyrieu opened the pull request:
Added back-end for user bookmarks, created new AJAX student model and added a function to bookmark jobs/listings to the interface.

cyrieu committed:
Initial commit

cyrieu committed:
Fixed bugs with ajax call

Removed $params because I don’t know how it works, forgot to update
header.php so I added the controller to the list

coollog commented on app/controllers/StudentControllerAJAX.php:
```diff
@@ -0,0 +1,43 @@
+
+      // Takes POST data from job/sublet page and adds it to student's bookmarks
+      // $_POST will contain: id, type, title
+      public static function bookmark() {
```
Should rename this (and route) to like addBookmark, and should have function for removing bookmarks and getting a list of bookmarks in json form as well.

cyrieu committed:
Added extra CRUD methods to bookmarking backend

Implemented remove bookmark functionality and getting a list of
bookmarked items in JSON

Added routes to route.php

coollog commented:
@cyrieu lol

cyrieu commented:
Hahaha finally closing this
On Aug 20, 2019, 2:31 PM -0700, Q Chen <notifications@github.com>, wrote:
> @cyrieu lol
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
